tests/e2e/scripts/gke/deploy.sh

```diff
@@ -121,6 +122,40 @@ fi
 run kubectl create -f ${YAML_FILE} --namespace "${NAMESPACE}"
 HOST=$(get_cluster_host "${NAMESPACE}")
 
+# Run in background while e2e tests are running.
+# ESPv2 is deployed in managed mode for all these e2e tests.
+# This will cause ESPv2 to rebuild the Envoy listener while lots of traffic is running through.
+function doServiceRollout() {
+  if [ "${BACKEND}" != 'bookstore' ]; then
+    echo "doServiceRollout: TODO(nareddyt): Support managed service rollout for ${BACKEND}"
+    return 0
+  fi
+
+  while true; do
+    echo 'doServiceRollout: Sleeping until next service rollout'
+    sleep 15m
+
+    local allow_cors=''
+    if (( RANDOM % 2 )); then
+      allow_cors=true
+    else
+      allow_cors=false
+    fi
+
+    echo "doServiceRollout: Setting allowCors = ${allow_cors}"
+    local tmp_file=$(mktemp)
+    cat "${SERVICE_IDL}" \
+        | jq ".\"x-google-endpoints\"[0].allowCors = $allow_cors" \
+        > "${tmp_file}"
+    mv -f "${tmp_file}" "${SERVICE_IDL}"
```

Comment:
not need to change service config.  We don't have anything to compare them, everytime if there is a "deploy",  a new config_id and rollout_id is generated.

Reply:
Agreed as mentioned in PR description. I guess I should clarify, I'm trying to kill 2 birds with 1 stone in this PR:
1. Do a service rollout while traffic is running
2. Test SC Reports for `allowCors`. We generate custom operation names for the CORS methods we generate. I want to verify they show up in the Endpoints Console, and if they'll be formatted correctly.

I can remove the modifying `allowCors` every time before deploy, but I find it helpful for debugging. Then I can look at the prow job logs and verify that Config Manager picked up the new rollout. Otherwise, we have no logging in our prow job to verify this, we have to look at the container logs.

Reply:
I see.   Or if we can find a string field in openapi spec that we can modify for each change. 

We should be able to get the config_id It is using from envoy admin port or from config_mgr, verify the new config id is picked up correctly.

Reply:
It's hard to do a verification on the config id. We don't expose this anywhere easily accessible. Even in the envoy admin port, the only place we could view this is the `/config_dump`. This requires parsing the entire listener config to find the config id in the SC filter.

I think just having some client-facing behavior change is good enough. In this case, the response code changes when we make an OPTIONS request based on the current config:
- `allowCors: true` will let the backend response with 200s
- `allowCors: false` will cause ESPv2 to respond with 404

Reply:
SImplified the test, remove all the cors related items. No need for verification, our integration test do this. We just want to see if there are any crashes.